python/interpret_community/common/gpu_kmeans.py

```diff
@@ -50,6 +51,15 @@ def kmeans(X, k, round_values=True):
         raise RuntimeError(
             "cuML is required to use GPU explainers. Check https://rapids.ai/start.html \
             for more information on how to install it.")
+    if cuml.__version__ >= '21.06':
+        from cuml.explainer.sampling import kmeans_sampling
+        summary, group_names, labels = kmeans_sampling(X, k, round_values, detailed=True)
+
+        return DenseData(summary,
```

Comment:
hmm I wish there was some way to add tests for these to ensure they are working correctly, maybe I need to look into getting GPU VMs for running cuML functionality

Reply:
Adding versioned tests in pytest might help, but yes a second set of eyes to double check it works would be really useful!